src/Constants/Message.js

```diff
@@ -0,0 +1,25 @@
+export const SYSTEM_MESSAGE = Object.freeze({
+  INPUT_PURCHASE_AMOUNT: '구입금액을 입력해 주세요.\n',
+  PURCHASE_LOTTO_MESSAGE: '개를 구매했습니다.',
+  INPUT_GOAL_NUMBER: '당첨 번호를 입력해 주세요.\n',
+  INPUT_BONUS_NUMBER: '보너스 번호를 입력해 주세요.\n',
+});
+
+export const resultMessage = (correctArray, rateOfReturn) =>
+  `당첨 통계
+  ---
+  3개 일치 (5,000원) - ${correctArray[0]}개
+  4개 일치 (50,000원) - ${correctArray[1]}개
+  5개 일치 (1,500,000원) - ${correctArray[2]}개
+  5개 일치, 보너스 볼 일치 (30,000,000원) - ${correctArray[3]}개
+  6개 일치 (2,000,000,000원) - ${correctArray[4]}개
+  총 수익률은 ${rateOfReturn}%입니다.`;
+
+export const ERROR_MESSAGE = Object.freeze({
+  IS_EMPTY: '[ERROR] 구입금액을 0원 이상 입력하세요.',
+  INPUT_NUMBER: '[ERROR] 숫자를 입력하세요.',
```

Comment:
```suggestion
errorString = (message) => `[ERROR] ${message}`
  INPUT_NUMBER: errorString('숫자를 입력하세요.')
```
반복되는 [ERROR] 스트링을 별도의 함수를 통해 붙여줄 수 있을 것 같아요!
아니면 아예 하나의 클래스로 선언해서 에러 객체를 만드시는 분도 계시더라구요